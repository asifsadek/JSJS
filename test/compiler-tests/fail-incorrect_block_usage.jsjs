val x = "outer";

val res = {
  print(x);
  val x = "inner";
  print(x);
};
