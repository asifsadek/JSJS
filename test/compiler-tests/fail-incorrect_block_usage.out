Error: There was a syntax error in your file.
